GVRf/Framework/src/org/gearvrf/GVRCameraRig.java

```diff
@@ -156,6 +159,8 @@ public float getFloat(String key) {
      *            The {@code float} value to map.
      */
     public void setFloat(String key, float value) {
+        checkKeyNotNullOrEmpty(key);
```

Comment:
This particular `String` is a key - but there might well be cases where you don't want a null-or-empty value. Suggest changing method to

    void checkStringNotNullOrEmpty(String name, String value) throws // ...